docs/DeveloperGuide.md

```diff
@@ -269,25 +268,26 @@ The following activity diagram summarizes what happens when the `whoIsFree` comm
 
 ### View Schedule Feature
 
+Views a person's schedule that will be displayed in the right panel.
+
 #### Implementation
 
 To allow users to view their contact's schedules, we implemented a `ViewScheduleCommand`, and added a `FilteredList<Person>` object in `ModelManager` to facilitate its execution.
 
 Moreover, we created `ScheduleCard.java`, `ScheduleCardPanel.java`, and their respective `.fxml` files so it will be easier to maintain or develop the GUI in the future.
```

Comment:
Actually now that I read, the java files and fxml files dont make it easier to maintain or develop the GUI right, it is just necessary to have it in order for the view to work?